lib/tasks/common_passwords_tasks.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+namespace :decidim do
+  namespace :common_passwords do
+    desc "Update common passwords list"
+    task :update do
+      urls = %w(
```

Comment:
Could we move this update logic to `Decidim::CommonPasswords` and just call the `update_dict!` method (for instance) here?